CodePush.m

```diff
@@ -161,6 +161,11 @@ - (void)dealloc
     [[NSNotificationCenter defaultCenter] removeObserver:self];
 }
 
+- (NSString *)getDeploymentKeyFromStatusReportIdentifier:(NSString *)statusReportIdentifier
```

Comment:
What do you think about moving all these report status methods out into their own class? The CodePush class is getting kind of cluttered and I'd like to start refactoring it out a little bit more.